client/src/components/application/ResponsiveDrawer/style.js

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  list: {
+    margin: '1rem 0',
+    padding: '0.5rem 2rem',
```

Comment:
If you go back to design you will see that there is no margin between the tabs titles
so I suggest changing these two lines to 
```suggestion
    padding: '1rem 2rem',
```